sdks/python/src/opik/api_objects/experiment/helpers.py

```diff
@@ -59,3 +65,19 @@ def get_experiment_data_by_name(
         for experiment in experiment_page_public.content:
             if experiment.name == name:
                 return experiment
+
+
+def check_prompt_args(
```

Comment:
this function is not just checking the arguments, it tries to handle them in a way to return a list of prompts